base.php

```diff
@@ -1163,7 +1163,7 @@ function ip() {
 		return isset($headers['Client-IP'])?
 			$headers['Client-IP']:
 			(isset($headers['X-Forwarded-For'])?
-				$headers['X-Forwarded-For']:
+				explode(',',$headers['X-Forwarded-For'])[0]:
```

Comment:
I also thought about it, but the schema is `, ` and luckily not ` ,` for separating IP addresses. I hope all servers respect this rule.